nemo_text_processing/text_normalization/token_parser.py

```diff
@@ -156,7 +156,8 @@ def parse_string_value(self) -> str:
 
         Returns parsed string value
         """
-        assert self.char not in string.whitespace and self.char != EOS
+        #assert self.char not in string.whitespace and self.char != EOS
+        assert self.char != EOS
```

Comment:
Why this change is needed?

Reply:
In Chinese, not every Chinese character has a space between them, and the spaces in a sentence have a special meaning that needs to be preserved.In this case,I have extended this part of the code